core/docker/docker-compose.yml

```diff
@@ -62,4 +64,4 @@ volumes:
   hapi-mysql-config:
     external: true
   instant:
-    external: true
+    external: true
```

Comment:
Add newline to end of file - some interpreters expect it and github complains about it